.buildkite/pipeline.yml

```diff
@@ -0,0 +1,91 @@
+env:
+  MAC_BK_AGENT: eng-prod-us-west-2-mac-arm-macos-build-medium
+  ENG_BK_AGENT: eng-default
+  FLUTTER_VERSION: 2.10.5
+  ANDROID_SDK_ROOT: ~/Library/Android/sdk
+  ANDROID_SDKMANAGER: ~/Library/Android/sdk/cmdline-tools/latest/bin/sdkmanager
+
+steps:
+  - name: ':hammer: Build and test iOS'
+    key: 'build-test-ios'
+    commands:
+      - . ~/.zshrc
+      - rbenv local 2.7.6
+      - brew tap leoafarias/fvm
+      - brew install fvm
+      - fvm install ${FLUTTER_VERSION}
+      - fvm use ${FLUTTER_VERSION}
+      - cd example
+      - xcrun simctl boot "iPhone 14" || true
+      - fvm flutter pub get
+      - fvm flutter precache --ios
+      - cd ios && pod install && cd -
+      - fvm flutter test
+      - fvm flutter test integration_test/app_test.dart
+    agents: 
+      queue: ${MAC_BK_AGENT}
+    timeout_in_minutes: 25
+
+  - name: ':hammer: Build and test Android'
+    key: 'build-test-android'
+    commands:
+      - . ~/.zshrc
+      - rbenv local 2.7.6
+      - brew tap leoafarias/fvm
+      - brew install fvm
+      - fvm install ${FLUTTER_VERSION}
+      - fvm use ${FLUTTER_VERSION}
+      - xcrun simctl shutdown "iPhone 14" || true
```

Comment:
Does this belong to `Build and test Android` step?

Reply:
Yeah just to 100% ensure ios sim is shutdown here since last step of `Build and test Android` still caught the ios simulator when I shut it down in the build-test-ios step for some reason